src/plugins/intel_cpu/tests/functional/single_layer_tests/interpolate.cpp

```diff
@@ -331,6 +337,8 @@ const std::vector<fusingSpecificParams> interpolateFusingParamsSet{
         emptyFusingSpec,
 #if defined(OPENVINO_ARCH_X86) || defined(OPENVINO_ARCH_X86_64)
         fusingSwish,
+        fusingAddPerTensor,
+        fusingMultiplyPerTensor,
```

Comment:
we need per-channel post-ops to cover this fix.

Reply:
let's remove this two lines as per-tensor post-ops is already covered, which doubled smoke test time. Interpolate smoke test and nightly test is heavy.